src/services/conversionRates/getConversionRatesService.js

```diff
@@ -292,14 +292,11 @@ const getConversionRates = async (app, requestedDate, symbol = 'ETH') => {
   const fiat = app.get('fiatWhitelist');
   const stableCoins = app.get('stableCoins') || [];
 
-  let token = getTokenBySymbol(app, symbol);
-  if (token.rateEqSymbol) {
-    token = getTokenBySymbol(app, token.rateEqSymbol);
-  }
+  const token = getTokenBySymbol(app, symbol);
 
   // This field needed for PAN currency
   const { coingeckoId } = token;
-  const requestedSymbol = token.symbol;
+  const requestedSymbol = token.rateEqSymbol || symbol;
```

Comment:
Merge latest develop to your branch and resolve conflicts... this part and previous one are changed

Reply:
I merged with develop and pushed again